lib/kubeclient/kube_exception.rb

```diff
@@ -9,6 +9,10 @@ def initialize(error_code, message, response)
   end
 
   def to_s
-    'HTTP status code ' + @error_code.to_s + ', ' + @message
+    string = "HTTP status code #{@error_code}, #{@message}"
+    if @response.is_a?(RestClient::Response) && @response.request
+      string << " for #{@response.request.method.upcase} #{@response.request.url}"
```

Comment:
@grosser should we protect in case any of these are ever `nil`?

This is a pretty dangerous part where to get `NoMethodError` exceptions. cc @cben